go/src/meguca/server/admin.go

```diff
@@ -262,10 +262,13 @@ func boardConfData(w http.ResponseWriter, r *http.Request) (
 // Handle requests to create a board
 func createBoard(w http.ResponseWriter, r *http.Request) {
 	var msg boardCreationRequest
+
```

Comment:
Seriously, why are you doing these?

Reply:
Everything being to close together triggers my autism, since it's less readable IMO.
If you really want them gone, I'll remove them.

Reply:
Nah, non-critical.

Reply:
Let me rephrase that: Do you want me to not do that from now on?

Reply:
Non-critical. I try to not be as opinionated.